app/build.gradle

```diff
@@ -16,8 +16,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 'android-P'
-    buildToolsVersion '28.0.0 rc1'
+    compileSdkVersion 28
```

Comment:
We should add `buildToolsVersion '28.0.0'` here.

Reply:
added